packages/spotify.rb

```diff
@@ -33,14 +33,8 @@ def self.install
     desktop_prefix = "#{CREW_DEST_PREFIX}/share/applications"
     binpath        = "#{CREW_DEST_PREFIX}/bin"
 
-    {
-      './usr/share'     => "#{CREW_DEST_PREFIX}/share"
-    }.each_pair do |src, dst|
-      FileUtils.mkdir_p dst
-      FileUtils.cp_r Dir["#{src}/*"], dst
-    end
-
-    FileUtils.mkdir_p [binpath, icon_prefix, desktop_prefix]
+    FileUtils.mkdir_p ["#{CREW_DEST_PREFIX}/share", binpath, icon_prefix, desktop_prefix]
+    FileUtils.cp_r Dir['usr/share/*'], "#{CREW_DEST_PREFIX}/share"
```

Comment:
Easier? `FileUtils.cp_r 'usr/share/', CREW_DEST_PREFIX.to_s`?

Reply:
Depending on the directory size, `FileUtils.mv` could work too and require less disk space to install.

Reply:
Switched to `FileUtils.mv`